src/DrCabal/Profile/Phase.hs

```diff
@@ -0,0 +1,11 @@
+module DrCabal.Profile.Phase
+  ( Phase (..)
+  )where
+
+data Phase = Phase
+    { phaseDownloading :: Word64
+    , phaseStarting    :: Word64
+    , phaseBuilding    :: Word64
+    , phaseHaddock     :: Word64
+    , phaseInstalling  :: Word64
+    }
```

Comment:
I think this module is redundant and the `Phase` data type should be moved to the `DrCabal.Profile.Stacked` module.

It's true that for the `Timeline` implementation the data type will have exactly the same number of fields with the same type. But those `Word64` would mean "timestamps" and not "duration". It's really confusing to have two datatypes that represent completely different semantic entities.

I imagine the `Timeline` module will have a similar data type but it'll be called `Timeline` instead.

Reply:
Done.